int16 chnl1
int16 chnl2
int16 chnl3
int16 chnl4
int16 chnl5
int16 chnl6
int16 chnl7
int16 chnl8
